cmake/developer_package/packaging/archive.cmake

```diff
@@ -90,6 +90,8 @@ macro(ov_define_component_include_rules)
     # scripts
     unset(OV_CPACK_COMP_INSTALL_DEPENDENCIES_EXCLUDE_ALL)
     unset(OV_CPACK_COMP_SETUPVARS_EXCLUDE_ALL)
+    # npu internal tools
```

Comment:
Hi @ilya-lavrenov! Could you please take a look at this part with cpack configs? 

Idea: npu_internal cpack target available in CI cache to be imported by NPU CI and NPU release build infrastructure for preparing extended packages.

Requirement: npu_internal must not be included in OpenVINO releases and dev drops. As I understand it is already controlled by product-configs in packaging/config.json#L381